### ISSUE TITLE
Unable to move .desktop file when installing an application

### ISSUE THREAD

**Issue by devadarta** [ivan-hc/AM#791]
SO: Archcraft
WM: i3wm
Error Message: mv: cannot move './zoom.desktop' to '/usr/local/share/applications/zoom-AM.desktop': No such file or directory
Cause: No such folder '../share/appllications' in /usr/local'
Suggestion: Create a folder in ~/.local/share/applications/ and move the file
Workaround to show the program in the menu (rofi): I manually copied the file /opt/zoom/zoom.desktop to ~/.local/share/applications

**Comment by ivan-hc** [comment]
Hi, is your version of AM updated?

Try `am -s` or `am -u`, and then install the app again.

I've solved this issue three days ago https://github.com/ivan-hc/AM/commit/425730cdf0f2a90bccb5da4b79b7c5b4c3190a21

The issue is because a mistake. In version 7 I removed the following command from the module "install.am"
```
$SUDOCOMMAND mkdir -p /usr/local/share/applications
```
it is needed to create the directory /usr/local/share/applications.
